lib/ecl/smspec_node.cpp

```diff
@@ -411,83 +387,69 @@ static void smspec_node_set_flags( smspec_node_type * smspec_node) {
    new timesteps you add after the change. Already created timesteps
    will not be updated if the default value is changed.
 */
-void smspec_node_set_default( smspec_node_type * smspec_node , float default_value) {
-  smspec_node->default_value = default_value;
-}
 
 
-float smspec_node_get_default( const smspec_node_type * smspec_node ) {
-  return smspec_node->default_value;
+void smspec_node_type::set_default( float default_value_) {
+  this->default_value = default_value_;
 }
 
-
-smspec_node_type * smspec_node_alloc_new(int params_index, float default_value) {
-  smspec_node_type * node = new smspec_node_type();
-
-  UTIL_TYPE_ID_INIT( node , SMSPEC_TYPE_ID);
-  node->params_index  = params_index;
-  smspec_node_set_default( node , default_value );
-
-  node->var_type      = ECL_SMSPEC_INVALID_VAR;
-  smspec_node_set_invalid_flags( node );
-  return node;               // This is NOT usable
+void smspec_node_set_default( smspec_node_type * smspec_node , float default_value) {
+  smspec_node->set_default(default_value);
 }
 
-
-static void smspec_node_set_wgname( smspec_node_type * index , const char * wgname ) {
-  index->wgname = wgname;
+float smspec_node_type::get_default() const {
+  return this->default_value;
 }
 
-
-
-static void smspec_node_set_lgr_name( smspec_node_type * index , const std::string& lgr_name ) {
-  index->lgr_name = lgr_name;
+float smspec_node_get_default( const smspec_node_type * smspec_node ) {
```

Comment:
Would be good if these small dummy functions which are only provided to guarantee a stable C api could be collected in one place - e.g. at the bottom of the file, or even in a separate file.